arch/xtensa/Kconfig

```diff
@@ -46,7 +46,7 @@ config XTENSA_USE_CORE_CRT1
 config XTENSA_ENABLE_BACKTRACE
 	bool "Backtrace on panic exception"
 	default y
-	depends on SOC_ESP32 || SOC_FAMILY_INTEL_ADSP
+	depends on SOC_SERIES_ESP32 || SOC_FAMILY_INTEL_ADSP
```

Comment:
`SOC_ESP32` -> `SOC_SERIES_ESP32` happens in `hal_espressif`, so updating west manifest should be atomic (in single commit) with this change.

Reply:
Updating the `west.yml` is not sufficient since it must be followed by `west update hal_espressif` call. This is true for any other PR involving the hal changes. Please take a look at the history.

Reply:
The assumption is that `west update` is called whenever you checkout on new commit. So for example when doing `git bisect` you do `west update` every time. Which means... that `west.yml` update needs to be in the same commit as `SOC_ESP32` -> `SOC_SERIES_ESP32` in the Zephyr codebase.